System.Device.I2c/I2cDevice.cs

```diff
@@ -11,25 +11,40 @@ namespace System.Device.I2c
     /// </summary>
     public class I2cDevice : IDisposable
     {
+        // this is used as the lock object 
+        // a lock is required because multiple threads can access the device
+        [Diagnostics.DebuggerBrowsable(Diagnostics.DebuggerBrowsableState.Never)]
+        private readonly object _syncLock;
 
-        private Windows.Devices.I2c.I2cDevice _device;
-        // For the ReadByte and WriteByte operations
-        private byte[] bufferSingleOperation = new byte[1];
+        [Diagnostics.DebuggerBrowsable(Diagnostics.DebuggerBrowsableState.Never)]
+        private readonly I2cConnectionSettings _connectionSettings;
+
+        [Diagnostics.DebuggerBrowsable(Diagnostics.DebuggerBrowsableState.Never)]
+        private bool _disposed;
+
+        // speeds up the execution of ReadByte and WriteByte operations
+        private readonly byte[] _buffer;
```

Comment:
This now should be SpanByte

Reply:
Can't do...
CS8345 Field or auto-implemented property cannot be of type 'SpanByte' unless it is an instance member of a ref struct.

Reply:
hum, then it makes less sense to do this. We can tweak SpanByte to be a ref struct (it's technically one on the Span<byte> world.